src/ds/d400/d400-device.cpp

```diff
@@ -1081,6 +1103,27 @@ namespace librealsense
         //TODO: Implement
     }
 
+    // Check if need change camera name due to number modifications on one device PID.
+    void update_device_name(std::string& device_name, const ds::d400_caps cap)
+    {
+        std::regex old_name_reg("D455");
+
+        switch (cap)
+        {
+        case ds::d400_caps::CAP_IR_FILTER:
+            device_name += "F"; // Adding "F" to end of device name if it has IR filter.
+            break;
+
+        case ds::d400_caps::CAP_IP65:
+            device_name = std::regex_replace(device_name, old_name_reg, "D456"); // Change device name from D455 to D456.
```

Comment:
Can we remove line 1109 and place here:
```
device_name = std::regex_replace(device_name, std::regex("D455"), "D456"); // Change device name from D455 to D456.
```
?
The parameter at line 1109 is only relevant on this case